src/plugins/backdropScreensaver/plugin.js

```diff
@@ -12,15 +12,15 @@ class BackdropScreensaver {
     }
     show() {
         const query = {
-            ImageTypes: 'Backdrop',
-            EnableImageTypes: 'Backdrop',
-            IncludeItemTypes: 'Movie,Series,MusicArtist',
+            ImageTypes: 'Backdrop,Backdrop1,Backdrop2,Backdrop3,Backdrop4,Backdrop5',
+            EnableImageTypes: 'Backdrop,Backdrop1,Backdrop2,Backdrop3,Backdrop4,Backdrop5',
```

Comment:
These image types don't exist, why add them?

Reply:
I was hoping someday it could work like this. I'd love to be able to have multiple backdrop images and it be able to use them each at random. 

I suppose atm there is no reason for it, but it doesn't seem cause issues to my client.

Reply:
Supporting multiple images of the same type doesn't require introducing of extra types.

Actual image retrieval is here:
https://github.com/jellyfin/jellyfin-web/blob/69bb25553efe7f0598786603acb25cbad088a334/src/components/slideshow/slideshow.js#L75
I haven't checked if `item.BackdropImageTags` contains more than one item.

Reply:
Okay I removed all the Backdrop# entries.

Reply:
> I haven't checked if `item.BackdropImageTags` contains more than one item.

Checked - it is filled with multiple backdrops.

If you want random backdrops, you need increase [this](https://github.com/jellyfin/jellyfin-web/blob/24929051b2b01f3c7ce1030b9afae5fe08524bd9/src/plugins/backdropScreensaver/plugin.js#L21) and change [this](https://github.com/jellyfin/jellyfin-web/blob/24929051b2b01f3c7ce1030b9afae5fe08524bd9/src/components/slideshow/slideshow.js#L74).

Reply:
Oh awesome thank you! I'd like to change the `ImageTypeLimit` to 5, but I'm not sure what to replace the `backdropImageTags[0]`; with. 
Could I use?
```
        const randomIndex = Math.floor(Math.random() * item.BackdropImageTags.length);
        options.tag = item.BackdropImageTags[randomIndex];
```

Reply:
It could be [`randomInt(0, item.BackdropImageTags.length - 1)`](https://github.com/jellyfin/jellyfin-web/blob/2d353e1845d9eecf5f9b8c58cfe9ebc8f3e00e6a/src/utils/number.ts#L11) but yours should work too.